dd-sdk-android/src/test/kotlin/com/datadog/android/rum/internal/domain/scope/RumViewScopeTest.kt

```diff
@@ -1352,7 +1403,7 @@ internal class RumViewScopeTest {
     }
 
     @Test
-    fun `𝕄 send event 𝕎 handleEvent(ApplicationStarted) on stopped view`(
+    fun `𝕄 close the scope 𝕎 handleEvent(ActionSent) on stopped view { ApplicationStarted }`(
```

Comment:
just a question: when this (`ApplicationStarted + stopped view`) situation happens?

Reply:
I agree, the application started cannot happen on a stopped view in theory… Then again, the reason why is outside of the scope of this class, so this test kinda makes sense

Reply:
Like @xgouchet explained below, this is just to make sure we test all the scenarios in this class.